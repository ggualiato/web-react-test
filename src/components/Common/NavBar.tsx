import React, { FC } from "react";

const NavBar: FC = () => {
    return (
        <nav>
            <span>Fliper Test</span>
        </nav>
    );
};

export default NavBar;
