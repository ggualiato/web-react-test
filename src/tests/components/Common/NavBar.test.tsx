import React from "react";
import { shallow } from "enzyme";
import NavBar from "../../../components/Common/NavBar";

test("should render NavBar correctly", () => {
    const wrapper = shallow(<NavBar />);
    expect(wrapper).toMatchSnapshot();
});
