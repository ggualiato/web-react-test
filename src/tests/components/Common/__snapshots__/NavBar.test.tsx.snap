// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NavBar correctly 1`] = `ShallowWrapper {}`;
